core/chains/evm/config/chain_specific_config.go

```diff
@@ -332,6 +332,23 @@ func setChainSpecificConfigDefaultSets() {
 	okxMainnet := fallbackDefaultSet
 	okxTestnet := okxMainnet
 
+	// Metis is an L2 chain based on Optimism.
+	metisMainnet := fallbackDefaultSet
+	metisMainnet.balanceMonitorBlockDelay = 0
+	metisMainnet.blockHistoryEstimatorBlockHistorySize = 0 // Force an error if someone set GAS_UPDATER_ENABLED=true by accident; we never want to run the block history estimator on metis
+	metisMainnet.chainType = config.ChainMetis
+	metisMainnet.finalityDepth = 1    // Sequencer offers absolute finality
+	metisMainnet.gasBumpThreshold = 0 // Never bump gas on metis
+	metisMainnet.gasEstimatorMode = "L2Suggested"
+	metisMainnet.linkContractAddress = "TODO"
+	metisMainnet.minIncomingConfirmations = 1
+	metisMainnet.minGasPriceWei = *big.NewInt(0) // Metis uses the L2Suggested estimator; we don't want to place any limits on the minimum gas price
+	metisMainnet.minRequiredOutgoingConfirmations = 0
```

Comment:
Matches Optimism changes from #6621
```suggestion
	metisMainnet.minRequiredOutgoingConfirmations = 1
```